app/controllers/journal_entries_controller.rb

```diff
@@ -23,6 +23,26 @@ def create
     end
   end
 
+  def show
+    @journal_entry = JournalEntry.find(params[:id])
+  end
+
+  def edit
+    @journal_entry = JournalEntry.find(params[:id])
+  end
+
+  def update
+    @journal_entry = JournalEntry.find(params[:id])
```

Comment:
this line is repetitive. please DRY it.

Reply:
Thank you for pointing it out i added a before action, load journal entry method